tests/experiments/test_language_strategies.py

```
@@ -252,7 +252,7 @@ def print_report(self, analysis: Dict):
         print(f"  Languages: 3 (English, Japanese, Spanish)")
         print(f"  Runs per combination: 5")
         print(f"  Total tests: {analysis['total_tests']}")
-        print(f"  Model: gemma3:12b-it-qat (Ollama)")
+        print(f"  Model: gemma3:12b (Ollama)")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)